mozilla/bedrock
# Bug 1392501 homepage experiment q32017

## Description

**Analytics requirements yet to come.**

Adds homepage variation that will be served to 100% of en-US visitors when switch is enabled.

## Issue / Bugzilla link

https://bugzilla.mozilla.org/show_bug.cgi?id=1392501

## Testing

https://bedrock-demo-jpetto-home.us-west.moz.works/

To test on the demo, ensure the above URL is added to `browser.uitour.testingOrigins` on `about:config`. ([docs](http://bedrock.readthedocs.io/en/latest/uitour.html#local-development))

To access the different promos:

1. Use Firefox signed in to Sync to see the Pocket promo
2. Use Firefox signed out of Sync *OR* Firefox on iOS/Android to see the Sync promo
3. Use non-Firefox to see the download Firefox promo

Ensure rendering is good in all browsers (IE 9+, Safari, Chrome).
